x-pack/test/functional/apps/ml/data_visualizer/index_data_visualizer_actions_panel.ts

```diff
@@ -58,5 +58,32 @@ export default function ({ getService }: FtrProviderContext) {
         await ml.jobWizardAdvanced.assertDatafeedQueryEditorValue(advancedJobWizardDatafeedQuery);
       });
     });
+
+    describe('view in discover page action', function () {
```

Comment:
I haven't tested yet, but this action should probably also be available with a `basic` license. The advanced wizard link is not, which is why this test file has been extracted from the `index_data_visualizer` test: it's only executed on `trial` licenses. So if the discover link is available in `basic`, we should probably split this file again into something like `index_data_visualizer_actions_panel_trial` and `index_data_visualizer_actions_panel_basic`.

Reply:
Updated here https://github.com/elastic/kibana/pull/89471/commits/60d483d54f40cbf5ab8da684a526a5c2ebef10e1